src/NuProj.ProjectSystem.12/Templates/Projects/NuProj/NuProj.nuproj

```diff
@@ -32,4 +32,10 @@
     <Content Include="Readme.txt" />
   </ItemGroup>
   <Import Project="$(NuProjPath)\NuProj.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <Target Name="BeforeBuild">
+  </Target>  
+  <Target Name="AfterBuild">
+  </Target>
+  -->
```

Comment:
I agree with the purpose. I just feel like it contributes to noise for almost everyone, and the few who want to customize the build *should* know enough about MSBuild to not need the hint. The cost outweighs the benefits by my estimation.